packages/updatePrescriptionStatus/tests/testUpdatePrescriptionStatus.test.ts

```diff
@@ -228,4 +228,38 @@ describe("buildDataItems", () => {
     expect(dataItems[0].PrescriptionID).toEqual(prescriptionID)
     expect(dataItems[0].PharmacyODSCode).toEqual(pharmacyODSCode)
   })
+
+  it("should include RepeatNo in data item when defined", () => {
+    const task = validTask()
+    const lineItemID = v4().toUpperCase()
```

Comment:
You don't need lines 234-236 or 239-241. Those are only needed in the above test.

Reply:
The code has been cleaned up.